README.md

```diff
@@ -11,7 +11,7 @@ import 'airbnb-js-shims';
 ## Included shims
 
  - [es5-shim](https://www.npmjs.com/package/es5-shim)
- - [es5-sham](https://www.npmjs.com/package/es5-shim)
+ - [es5-shim](https://www.npmjs.com/package/es5-shim)
```

Comment:
I believe this was intentionally `es5-sham`. But perhaps the link should point to here? https://www.npmjs.com/package/es5-shim#shams

Reply:
indeed, it's intentional, and that's a better link for it to point to.
```suggestion
 - [es5-sham](https://www.npmjs.com/package/es5-shim#shams)
```